spacy_llm/tasks/rel/task.py

```diff
@@ -160,3 +113,11 @@ def _cfg_keys(self) -> List[str]:
             "_label_definitions",
             "_verbose",
         ]
+
+    def scorer(self, examples: Iterable[Example]) -> Dict[str, Any]:
+        # todo how to score REL?
+        raise NotImplementedError
```

Comment:
Maybe we could add a utility function `is_scorable()` that can be called instead of having to `catch` this `NotImplementedError`.